src/Onboarding/FormRepository.php

```diff
@@ -85,7 +86,7 @@ protected function makeAndPersist(): int
     {
         $campaign = Campaign::create([
             'type' => CampaignType::CORE(),
-            'title' => __('GiveWP Onboarding', 'give'),
+            'title' => __('Fundraising Campaign Form', 'give'),
```

Comment:
I'm thinking we should drop the word _Form_ from the campaign name 😄 

```suggestion
            'title' => __('Fundraising Campaign', 'give'),
```